app/routes/overview/components/NextEvent.css

```diff
@@ -12,6 +12,13 @@
   margin: 0;
 }
 
+.info {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: -3px 10px;
```

Comment:
Try to avoid negative margins. I'm assuming this is for the while event text thing? Of so try to rather decrease the spacing thats created by some other css :)

Reply:
Not sure how to go about this. The negative margin makes the title of the event and the icons n stuff closer together, which looks nicer imo. The margin of the title and the padding of the eventItem is not set, so I can't see how I could drecrease the spacing elsewhere.